### PR TITLE
Care Quality Commission for England  I.P.C. 

### PR DESCRIPTION
**Pull request by martynlesbirel**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json [hunk 1]
```diff
@@ -0,0 +1,843 @@
...
+  "paths": {
+    "/providers": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Providers, organisations for health, social care services regulated by the CQC",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json:28]: @martynlesbirel All the summary values need to improved as they are vague and doesn't look like operation names.
You can refer [a connector](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/SAS%20Decisioning/apiDefinition.swagger.json#L37) in our open source repo for reference.

Documentation - [Summaries](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@martynlesbirel Thank you so much for fixing version and contact section, we will move ahead with deployment

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by mamurshe — CHANGES_REQUESTED** [review]
Please renamed the swagger to `apiDefinition.swagger.json`

**Comment by sriyen-msft** [comment]
[[certify-connector]]

File: independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json [hunk 2]
```diff
@@ -0,0 +1,844 @@
...
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "operationId": "Changes",
+        "summary": "List of organisations of specified type that have changed in given timeframe",
+        "description": "The Changes service provides a mechanism of staying informed about updates to providers and locations, to allow users to refresh their cached data if necessary Changes for two organisation types are available: provider or location API consumers must specify a start and end date range, see the API reference for more detail This service allows identification of which organisations have changed over time For successive calls to this service, the startTime parameter should be set to the endTime of the previous call The times correspond to the time that this service itself acquires the data and is not, for example, the time of data entry onto a source system",
```
Comment (ptalrejagithub) [inline independent-publisher-connectors/Care Quaility Comission For England/CQC-Connector.swagger.json:632]: Can you please update the string? It has missing periods in the whole segment.

**Pull request merged**